app/src/containers/Donor/Donor.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as getDonor from './donorActions';
+
+export class Donor extends React.Component {
+
+  render() {
+
+    let abilityTopresent = (this.props.donor.is_available == null ||
```

Comment:
You can extract the parameter is_available from this.props.donor, instead writing twice  this.props.donor